parachain/runtime/interlay/src/lib.rs

```diff
@@ -1583,6 +1584,7 @@ impl_runtime_apis! {
             use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey};
             impl frame_system_benchmarking::Config for Runtime {}
             // impl  runtime_common::benchmarking::orml_tokens::Config for Runtime {}
+            // impl  runtime_common::benchmarking::orml_vesting::Config for Runtime {}
```

Comment:
Is should be possible to include this for the `interlay` runtime. The only reason we did not include `orml-tokens` yet is because the runtime does not contain the lending components.